src/main/java/org/folio/service/storage/LocalFileStorageService.java

```diff
@@ -72,8 +72,14 @@ public String getServiceName() {
   public Future<Boolean> deleteFile(FileDefinition fileDefinition) {
     Promise<Boolean> promise = Promise.promise();
     try {
-      fs.deleteBlocking(fileDefinition.getSourcePath());
-      promise.complete(true);
+      String filePath = fileDefinition.getSourcePath();
+      if (fs.existsBlocking(filePath)) {
+        fs.deleteBlocking(filePath);
+        promise.complete(true);
+      } else {
+        LOGGER.warn("Couldn't detect the file with id {} in the storage: ", fileDefinition.getId());
```

Comment:
Seems need to delete symbol ":" from end of the log line

Reply:
Removed it, thank you!